Library/Homebrew/formula.rb

```diff
@@ -3215,6 +3215,8 @@ def link_overwrite_paths
 
     # Permit links to certain libraries that don't exist. Available on Linux only.
     def ignore_missing_libraries(*)
+      return if Homebrew::SimulateSystem.linux?
+
       raise FormulaSpecificationError, "#{__method__} is available on Linux only"
```

Comment:
👍🏻 to ship as-is but it makes me wonder if we'll need any more of the `extend/linux` code for accurate results.

Reply:
Agreed. I pushed a fix but then decided to split that off into a separate PR. Merging so there's no error, but will clean this up more.

Reply:
Wah Oh!